discovery_engine/lib/src/domain/models/market/feed_market.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:equatable/equatable.dart';
```

Comment:
could we move this file into parent folder => `discovery_engine/lib/src/domain/models/feed_market.dart`? all the other files in `domain/models` are at the same level, would be good to have consistency

Reply:
also please add licence notice
```
// Copyright 2022 Xayn AG
//
// This program is free software: you can redistribute it and/or modify
// it under the terms of the GNU Affero General Public License as
// published by the Free Software Foundation, version 3.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU Affero General Public License for more details.
//
// You should have received a copy of the GNU Affero General Public License
// along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Reply:
Thanks!

Reply:
Sure, if youd like - I will.
But in my personal opinion, `models` become a messy place, and it's better to split those files into separate folders (so all `feed_market` related in one folder and all `configuration` related - in another.)

But sure, you are right, this is not a part of this pr, so I will put it back